src/main/java/core/basesyntax/model/FruitRecord.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax.model;
+
+public class FruitRecord {
```

Comment:
Don't begin class implementation with an empty line. Remove the redundant empty line at the beginning of the class.